compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -6067,9 +6080,9 @@ private DiagnosticErrorCode validateArgumentOrder(SyntaxKind prevArgKind, Syntax
         return errorCode;
     }
 
-    private boolean isMissingPositionalArg(STNode arg) {
+    private boolean isPositionalArgWithSimpleNameRef(STNode arg) {
         STNode expr = ((STPositionalArgumentNode) arg).expression;
-        return expr.kind == SyntaxKind.SIMPLE_NAME_REFERENCE && ((STSimpleNameReferenceNode) expr).name.isMissing();
+        return expr.kind == SyntaxKind.SIMPLE_NAME_REFERENCE;
```

Comment:
Can we write this inline without using a method 
`((STPositionalArgumentNode) arg).expression.kind == SyntaxKind.SIMPLE_NAME_REFERENCE`